pkgs/data/fonts/google-fonts/default.nix

```diff
@@ -18,10 +20,6 @@ stdenv.mkDerivation rec {
     chmod -x $dest/*.ttf
   '';
 
-  outputHashAlgo = "sha256";
-  outputHashMode = "recursive";
-  outputHash = "0q03gg0sh2mljlbmhamnxz28d13znh9dzca84p554s7pwg6z4wca";
```

Comment:
I tried the following:
```patch
diff --git a/pkgs/data/fonts/google-fonts/default.nix b/pkgs/data/fonts/google-fonts/default.nix
index 802c287..fc91198 100644
--- a/pkgs/data/fonts/google-fonts/default.nix
+++ b/pkgs/data/fonts/google-fonts/default.nix
@@ -9,8 +9,15 @@ stdenv.mkDerivation rec {
     repo = "fonts";
     rev = "7a4070f65f2ca85ffdf2d465ff5e095005bae197";
     sha256 = "0c20vcsd0jki8drrim68z2ca0cxli4wyh1i1gyg4iyac0a0v8wx3";
+    extraPostFetch = ''
+      find $out -print0 | xargs -0 touch -t 197001010000
+    '';
   };
 
+  outputHashAlgo = "sha256";
+  outputHashMode = "recursive";
+  outputHash = "1dj3z9l3bcvs7hbpvmd1lppwzxsbfg9lilvkmn5h5wxfmragywpy";
+
   phases = [ "unpackPhase" "installPhase" ];
 
   installPhase = ''
```
and it does seem to work. I tried to rebuild from scratch (including download from GitHub) a few times and it successfully installed each time. Without these changes I do indeed get failures when `outputHash` is used.